tests/fixtures/config/env/includes/engine_location.yml

```diff
@@ -1,4 +1,10 @@
 
 engine.location:
   type: dev
-  path: '{PIPELINE_CONFIG}/config/bundles/test_app'
+  path: '{CONFIG_FOLDER}/bundles/test_engine'
+
+incomplete_engine.location:
```

Comment:
There is a test that relied on file copying to be able to rename files on disk. Since configs are read from the fixtures by default, this modified the fixtures folder, so instead I've added a broken engine with no startup.py